docs/index.rst

```diff
@@ -10,7 +10,8 @@ PlasmaPy Documentation
    :hidden:
 
    notebooks/physics
-   notebooks/dispersion_function
+   notebooks/dispersion/dispersion_function
+   notebooks/dispersion/two_fluid_dispersion
```

Comment:
```suggestion
   notebooks/dispersion/two_fluid_dispersion
```

We want to keep the number of notebooks on the RTD landing page to 3.  We're treating them like the feature cards on plasmapy.org.